Blizzard.IO.RabbitMQ.Builders/Rpc/NetqRabbitRpcBuilder.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Blizzard.IO.Core.Rpc;
+using Microsoft.Extensions.Logging;
+using Blizzard.IO.RabbitMQ.Rpc;
+using Blizzard.IO.RabbitMQ.Entities.Rpc;
+using System;
+
+namespace Blizzard.IO.RabbitMQ.Builders.Rpc
+{
+    public class NetqRabbitRpcBuilder : BaseNetqRabbitRpcBuilder
+    {
+        public NetqRabbitRpcBuilder(ILoggerFactory loggerFactory) : base(loggerFactory)
+        {
+        }
+
+        public NetqRabbitRpcBuilder AddRpcMessageType(RpcMessageType rpcMessageType)
+        {
+            RpcMessageType = rpcMessageType;
+            return this;
+        }
+
+        public NetqRabbitRpcBuilder AddRabbitConnectionProperties(
+            ushort timeout = 10,
+            string product = null,
+            string platform = null,
+            string virtualHost = null,
+            int requestHeartbeat = 10,
+            int prefetchCount = 50,
+            bool publisherConfirms = false,
+            bool persistentMessages = true)
+        {
+            Timeout = timeout;
+            PrefetchCount = prefetchCount;
+            Product = product;
+            Platform = platform;
+            PublisherConfirms = publisherConfirms;
+            RequestHeartbeat = requestHeartbeat;
+            VirtualHost = virtualHost;
+            PersistentMessages = persistentMessages;
+            return this;
+        }
+
+        public NetqRabbitRpcBuilder AddCredentials(string hostname, string username, string password)
```

Comment:
I think that you need a AddHost Method AddCredentials that get user and pass, and AddCredentialsAndHost Method.

Reply:
I think it will be better to split this function into 3, each on will take care of single parameter

Reply:
I could maybe agree on the splitting to two but it is still seems redundant.
If you add password or username you have to add the other one.
About the hostname it just seem redudnant not to pack those three together.